src/js/routes/TwitterHandleLanding.jsx

```diff
@@ -55,25 +55,25 @@ export default class TwitterHandleLanding extends Component {
 
   _onTwitterStoreChange () {
     // console.log("TwitterHandleLanding _onTwitterStoreChange");
-    let { twitter_followers_count } = TwitterStore.get();
+    let { twitterFollowersCount } = TwitterStore.get();
     const {
-      kind_of_owner, owner_we_vote_id, twitter_handle, twitter_description,  twitter_name,
-      twitter_photo_url, twitter_user_website, status,
+      kindOfOwner, ownerWeVoteId, twitterHandle, twitterDescription,  twitterName,
+      twitterPhotoUrl, twitterUserWebsite, status,
```

Comment:
Since these variables come from the store (which is still using lower case variables with underscores), the way the Twitter store holds the variable names needs to be mapped to our local variable. The format seems odd at first because it seems backwards, but it is like this (I just used the first two variables as an example):
```
const {
      kind_of_owner: kindOfOwner, owner_we_vote_id: ownerWeVoteId
    } = TwitterStore.get();
```